cli/command/user/update.go

```diff
@@ -95,7 +95,7 @@ func verifyGroupLogic(opt updateOptions) error {
 	return nil
 }
 
-func verifyUpdate(storageosCli *command.StorageOSCli, opt updateOptions) error {
+func verifyUpdate(opt updateOptions) error {
 	if !(opt.username == "" || verifyUsername(opt.username)) {
```

Comment:
I guess this is were the empty username issue is

Reply:
Yes, this is where it's even proceeding without a username for user update.

Reply:
Just realized that the `username` in options is used when the username is being updated. The argument parser would show error if username argument is not passed to the command.

Reply:
I tried updating username of an existing user and it's not allowed by CP. I think it's better just remove `username` from the options and this whole `if` condition.